### ISSUE TITLE
Allow array of button ID's on `useNavigationButtonPress`

### ISSUE THREAD

**Issue by karlmarxlopez** [underscopeio/react-native-navigation-hooks#13]
Example usage

```ts
useNavigationButtonPress(()=> {}, 'componentId', ['foo', 'bar']);
```

I can submit a PR if you like.

**Comment by jpgarcia** [comment]
Instead of using an array you can easily declare two hooks like this:

```js
 useNavigationButtonPress(() => {}, componentId, 'foo')
 useNavigationButtonPress(() => {}, componentId, 'bar')
```

**Comment by karlmarxlopez** [comment]
That's a lot of keystrokes. Array of buttonIds solves that issue, and it is specially useful when grouping related buttons into one listener @jpgarcia

**Comment by jpgarcia** [comment]
If you want to handle multiple button events you can do it like this (same way with the array approach):

```js
useNavigationButtonPress(e => {
  switch (e.buttonId) {
    case 'foo':
      // handler
      return 
    case 'bar':
      // handler
      return
    default:
      return
  }
}, componentId)
```

This Closes #14
